.github/workflows/main.yml

```diff
@@ -38,7 +38,13 @@ jobs:
       run: make -C scripts test
 
     - name: Test crit
-      run: sudo make -C test crit-test
+      run: |
+        make -C crit update-proto branch=${{ matrix.criu_branch }}
+        # If the definition of the protobuf files has changed upstream
+        # we just skip this test. Changed protobuf files will mean
+        # that crit-test will fail.
+        if [[ $(git status --porcelain | wc -l) -gt 0 ]]; then echo "Skipping crit-test because criu upstream changed protobuf definitions"; exit 0; fi
```

Comment:
Hmm, I'm not sure if this is a good idea.

We could just run `make gen-proto` and `crit-test` would pass even when the protobuf files have changed in the upstream repository:
```bash
make -C crit update-proto branch=${{ matrix.criu_branch }}
make -C crit gen-proto
sudo make -C test crit-test
```

Reply:
Good idea

Reply:
It was a good idea, but the protobuf version in Ubuntu 20.04 and 22.04 is too old to work with the files we generated with newer protobuf versions. Need to figure out how to best solve this.